.github/workflows/build.yml

```diff
@@ -51,7 +51,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ macos-latest, ubuntu-latest ]
+        os: [ macos-latest, ubuntu-latest, ubuntu-24.04-arm ]
```

Comment:
Technically `ubuntu-24.04-arm` is still in public preview, but it seems to work.

Reply:
We used to do a [manylinux](https://github.com/pypa/manylinux) build that magically supports all kinds of linux version and distributions. It would be nice to get back to that somehow.